api/v1beta2/image_config.go

```diff
@@ -29,16 +29,20 @@ import "fmt"
 type ImageConfig struct {
 	// Version is the version of FoundationDB this policy applies to. If this is
 	// blank, the policy applies to all FDB versions.
+	// +kubebuilder:validation:MaxLength=20
 	Version string `json:"version,omitempty"`
 
 	// BaseImage specifies the part of the image before the tag.
+	// +kubebuilder:validation:MaxLength=20
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxLength=200
```

The base image can include a registry.